server/graphql/queries.js

```diff
@@ -915,6 +915,46 @@ const queries = {
         total,
       };
     }
+  },
+
+  activeCollectives: {
+    type: CollectiveSearchResultsType,
+    description: 'An opinionated list of Collective with a minimum number of backers',
+    args: {
+      backerCount: {
```

Comment:
is that the only difference with the `allCollectives` query?
If so, cannot we merge the two to reduce the query surface?
Maybe also rename `backerCount` to `minBackerCount`? (the former makes conveys that you will be returning collectives with that exact number of backers)